Plugins/Feat/Feat.cpp

```diff
@@ -330,6 +330,8 @@ void Feat::ApplyFeatEffects(CNWSCreature *pCreature, uint16_t nFeat)
                     &CNWSCreature::CalculateSpellSaveDC,
                     +[](CNWSCreature *pThis, int32_t nSpellID) -> int32_t
                     {
+                        auto pSpell = Globals::Rules()->m_pSpellArray->GetSpell(nSpellID);
```

Comment:
When using `auto` please explicitly prefix pointers with a `*`, `auto *pSpell = ...` in this case.

You should also check it isn't returning a nullptr!

Reply:
Fixed